source/gps_questions.md

```diff
@@ -0,0 +1,134 @@
+| geopoint | point  | Collect the GPS coordinates of the house                    |
+| geotrace | road   | Collect the GPS coordinates of the road                     |
+| geoshape | garden | Collect the GPS coordinates of the boundaries of the garden |
+| survey   |
+
+## Appearance of “Point”, “Line” and “Area” question types in web forms and KoboCollect
+
+### Default appearance
+
+![Default appearances of GPS questions](images/gps_questions/gps_default_appearances.png)
+
+## Collecting GPS points in the background
+
+Besides adding GPS questions on the form, you can also enable the ability to
+collect GPS coordinates in the background as data is being collected. This is
+possible by turning on the “Audit” option in the Form Builder (Layout & Settings
+-> Meta questions) or by adding the `audit` meta question to your XLSForm. Learn
+more about how to do this [here](audit_logging.md).
+
+## Calculating distance and area with “Line” and “Area” question types
+
+As you collect your GPS data, you might need to calculate the distance and area
+from your “Line” and “Area” questions.
+
+### Calculating distance from “Line” questions
+
+To calculate the distance from a “Line” question type in the formbuilder, use
+the "Calculate" question type and the `distance()` function as shown below:
+
+![Calculate distance](images/gps_questions/calculate_distance.png)
+
+<section class="note">
+
+In the example above, the question “Capture the GPS of the track” has been added
+as a “Line” question type. The “Data Column Name” in the question settings has
+been changed to “track”.
+
+The question <code>distance(${track})</code> has been added as a “Calculate”
+question. Its Data Column Name has been changed to “distance”. The distance from
+the calculation will be in Meters.
+
+The “Note” question is optional and has been added with the purpose of
+displaying the calculated distance on the form.
+
+</section>
+
+In XLSForm, you can do the same as follows:
+
+| type      | name             | label                              | calculation        |
+| :-------- | :--------------- | :--------------------------------- | :----------------- |
+| geotrace  | track            | Capture the GPS of the track       |                    |
+| calculate | distance         |                                    | distance(${track}) |
+| note      | display_distance | The distance is ${distance} Meters |                    |
+| survey    |
+
+### Calculating area from “Area” questions
+
+You can calculate the area from an “Area” question type using the "Calculate"
+question type and the `area()` function in the formbuilder as shown below:
+
+![Calculate distance](images/gps_questions/calculate_area.png)
+
+<section class="note">
+
+In the example above, the question “Capture the GPS of the boundaries” has been
+added as an “Area” question type. The “Data Column Name” in the question
+settings has been changed to “boundary”.
```

Comment:
You don't need to say "changed" as you are setting up the form from scratch and this implies that it was set to something else before and now you are making it something else. Rather use a word like "set" or just "the value is X".